test/python/test_with_schedules.py

```diff
@@ -190,7 +190,7 @@ def pr_verified_test(self, input_file_name, use_separate_algo_file=False):
 
     def pr_delta_verified_test(self, input_file_name, use_separate_algo_file=False):
         if use_separate_algo_file:
-            self.basic_compile_test_with_separate_algo_schedule_files("pr_delta.gt", input_file_name)
+            self.basic_compile_test_with_separate_algo_schedule_files("cc_hybrid_dense_parallel_bitvector.gt", input_file_name)
```

Comment:
This line should not be changed. You should be writing a separate verified test function for closeness centrality.